examples/deployment/docker-compose/README.md

```diff
@@ -0,0 +1,8 @@
+
+
+```bash
+docker volume create --driver local \
```

Comment:
I think this readme needs more or removing, it doesn't make a whole load of sense at the moment.

Reply:
Yeah that was from a previous iteration where I was fucking with celery.